modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheAbstractSelfTest.java

```diff
@@ -150,8 +150,8 @@ else if (isMultiJvm() && !(storeStgy instanceof H2CacheStoreStrategy))
                                     jcache(fi).removeAll();
 
                                     if (jcache(fi).size(CachePeekMode.ALL) > 0) {
-                                        for (Cache.Entry<String, ?> k : jcache(fi).localEntries())
-                                            jcache(fi).remove(k.getKey());
```

Comment:
Is there a case when `GridCacheAbstractSelfTest#jcache` is preferable to `GridCommonAbstractTest#jcache`?

Reply:
GridCommonAbstractTest#jcache calls grid() and GridCacheAbstractSelfTest#jcache calls jcache(0) -> grid(0). So these methods calls different nodes.